### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by jcgasche**

### CODE REVIEW

File: people/jean-charles-gasche.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Jean-Charles Gasche
+------------------
+
+![](photos/jeancharlesgasche.jpg)
```
Comment (dblock) [inline people/jean-charles-gasche.md:4]: Rename the JPG to jean-charles-gasche.jpg, to be consistent, please.

**Comment by dblock** [comment]
Please also add a link from the README. Make sure you end up with 1 commit (you'll need to _squash_), see http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html if you need help.

**Comment by karnseth** [comment]
Hi Charles,

Feel free to get in touch if you need help also. My contact info is on the course page, and I'm also on Slack.
- Karn

**Comment by dblock** [comment]
I am closing this without merging, please reopen with ^^^. Thanks.

**Pull request closed without merging**